File: str_to_obj.js
var myString = "this is my home abdur rahman";


let myObject={};
function strToObj(data){
    for (let i=0; i<data.length; i++) {
        console.log(i);
     myObject[data[i]] = data[i].length;
     console.log(myObject);

    }
console.log(myObject);
return myObject;
}
var myArray = strToObj(myString.split(" "));
console.log("output", JSON.stringify(myArray))

File: str_to_obj.py
import json
myString = "this is my home abdur rahman";
obj = dict()
def strToObj(data):
    for i in range(len(data)):
        obj[data[i]] = len(data[i])
    return obj
   
mydata = strToObj(myString.split(" "))
print("OutPUT",json.dumps(mydata))

